src/index.html

```diff
@@ -0,0 +1,44 @@
+<form action="/" method="post" id="data" encType="multipart/form-data"></form>
+<table>
+    <tr>
+        <td>ALG HUD:</td>
+        <td><input type="number" name="hud" form="data"/></td>
```

Comment:
Can you use the following keys (extracted from `red-prefix-to-project-name.json`)?
```
{
  "ALGHUD": "ALG HUD",
  "COSTP": "Costpoint Integration",
  "FISAR": "FISAR",
  "FLTSTA": "Fleet Status",
  "FLTSTR": "Fleet Strategy",
  "RSCR": "Resource Mgmt & Optimization"
}
```

Then you could create the timesheet-info with:
```
const redPrefixToProjectName = require('../red-prefix-to-project-name.json');
...
timesheetInfo = {};
Object.keys(redPrefixToProjectName).forEach((prefix) => {
  const projName = redPrefixToProjectName[prefix];
  timesheetInfo[projName] = parseInt(req.body[prefix]) || 0;
})

```